src/api/app/controllers/person_controller.rb

```diff
@@ -30,7 +30,7 @@ def command
       internal_register
       return
     end
-    raise UnknownCommandError, "Allowed commands are 'change_password'"
+    raise UnknownCommandError, "Allowed commands are 'register'"
```

Comment:
Since there's only one command allowed, I suggest this:
```suggestion
    raise UnknownCommandError, "Allowed command is 'register'"
```

Reply:
Fixed